packages/logging/system-test/logging.js

```diff
@@ -347,6 +347,32 @@ describe('Logging', function() {
       });
     });
 
+    it('should preserve order of entries', function(done) {
+      var logEntries = [
+        log.entry('1')
+      ];
+
+      setTimeout(function() {
+        logEntries.push(log.entry('2'));
```

Comment:
I would suggest adding a third entry here perhaps with the same timestamp as '2'. In such a case the retrieval order should match the insertion order.